instance/models/mixins/openedx_storage.py

```diff
@@ -101,6 +95,13 @@ def _get_s3_settings(self):
             "AWS_S3_LOGS_SECRET_KEY": self.s3_secret_access_key,
         }
 
+        if self.s3_region:
+            s3_settings.update({
+                "aws_region": self.s3_region,
```

Comment:
Needed by https://github.com/edx/configuration/blob/open-release/hawthorn.1/playbooks/roles/aws/templates/send-logs-to-s3.j2#L107